libcxxabi/CMakeLists.txt

```diff
@@ -385,7 +388,7 @@ if (LIBCXXABI_SILENT_TERMINATE)
   add_definitions(-DLIBCXXABI_SILENT_TERMINATE)
 endif()
 
-if (LIBCXXABI_NON_DEMANGLING_TERMINATE)
+if (LIBCXXABI_NON_DEMANGLING_TERMINATE OR NOT LIBCXXABI_ENABLE_DEMANGLER)
```

Comment:
I generally support adding this capability, but I'd like to avoid making this change as an "addition of complexity", and instead modify what we already have. So it seems like we already have the ability to use a non-demangling `terminate()`. This was added by @DanAlbert a few years ago. Can we merge these two options? In other words, can we instead say that you get a non-demangling terminate if-and-only-if `LIBCXXABI_ENABLE_DEMANGLER=OFF`?

If that's reasonable, then we add this option but really we're not adding much complexity since we also remove something else.

Reply:
Thinking about it some more, I guess there's not really that much value in being able to control these separately. So I'd probably just use `LIBCXXABI_ENABLE_DEMANGLER` to control the demangling terminate as well, and remove the separate control for `LIBCXXABI_NON_DEMANGLING_TERMINATE`, like you suggested.